src/Sarif.Viewer.VisualStudio/Resources.resx

```diff
@@ -160,6 +160,20 @@
   <data name="RuleLookup" type="System.Resources.ResXFileRef, System.Windows.Forms">
     <value>Data\RuleLookup.json;System.Byte[], mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </data>
+  <data name="SaveTransformedV1LogFail_Access_DialogMessage" xml:space="preserve">
+    <value>The log file couldn't be saved because access to the path '{0}' was denied.</value>
+    <comment>{0} will be the full file path</comment>
+  </data>
+  <data name="SaveTransformedV1LogFail_General_Dialog" xml:space="preserve">
+    <value>The log file couldn't be saved: {0}</value>
+    <comment>{0} will be the exception message</comment>
+  </data>
+  <data name="SaveTransformedV1Log_DialogTitle" xml:space="preserve">
+    <value>Save Transformed Log File</value>
+  </data>
+  <data name="TransformV1_DialogMessage" xml:space="preserve">
+    <value>The log file you have opened is SARIF version 1. We will automatically transform this file to SARIF version 2. Would you like to save the transformed log file?</value>
```

Comment:
Check with a UI guru about saying "we". Perhaps "The log file will automatically be transformed to SARIF version 2...."